mxcube3/routes/SampleChanger.py

```diff
@@ -24,14 +29,20 @@ def get_samples_list():
     order = []
 
     for s in samples_list:
+        if not s.isPresent():
+            continue
+        if s.isLoaded():
+            state = SAMPLE_MOUNTED
+        else:
+            state = UNCOLLECTED
```

Comment:
This logic seems to indicate that SAMPLE_MOUNTED and UNCOLLECTED are mutually exclusive. Which is of course not the case since a sample can be SAMPLE_MOUNTED and UNCOLLECTED. Did you consider this case and are we just confused by the names of the states ?